tests/Main.hs

```diff
@@ -365,15 +365,22 @@ sinkEntrySpec =
 loadEntrySpec :: SpecWith (Path Abs File)
 loadEntrySpec =
   context "when an entry is loaded" $
-    it "is there" $ \path -> property $ \m b s -> do
+    it "is there" $ \path -> property $ \m b s r -> do
       let vpath = deriveVacant path
       B.writeFile (toFilePath vpath) b
+      addUTCTime (-60) <$> liftIO getCurrentTime >>= setModificationTime vpath
```

Comment:
19800101 is the earliest date ZIP supports. It is timezone-agnostic, so it's unclear whether people store UTC or local time. Also in order to avoid using 'zero' as a special test value, I bumped the constant date up to 19810101.